question-bank.json

```diff
@@ -196,12 +196,45 @@
             "How many hydrogen atoms are present in 1 mg of aspartame (C<sub>14</sub>H<sub>18</sub>N<sub>2</sub>O<sub>5</sub>), the artificial sweetener?": 19,
             "What is the mass, in grams, of all electrons within a single copper atom?": -26
         }
+        "example": {
```

Comment:
Hi, thanks for the merge request! Sorry, I didn't word it very well, so the idea with the sources is that you would put each group of questions in its own key associated with the source. So, if say your first 4 questions came from the Cornell source, you would put them in a question object called "Cornell" rather than "example". This helps associate which question came from which source. Sorry for the confusion!